server/resources/Skill.py

```diff
@@ -0,0 +1,53 @@
+from flask_restful import Resource
+from flask import request
+from marshmallow import ValidationError
+from models import db, ma, SkillModel, SkillSchema
+
+SKILL_ALREADY_EXISTS = "A skill with that name already exists."
+CREATED_SUCCESSFULLY = "Skill created successfully."
+SKILL_NOT_FOUND = "Skill not found."
+SKILL_DELETED = "Skill deleted."
+
+skill_schema = SkillSchema()
+skill_list_schema = SkillSchema(many=True)
+
+
```

Comment:
Same here, put all these different classes into the skill class. then instead of SkillCreate.post, have Skill.create and instead of SkillList.get, Skill.get_list